docs/user-guide/cli-updatingcli.md

```diff
@@ -1,98 +1,155 @@
+If you are running Zowe CLI version v2.0.0 to v2.15.x, you can update to `@zowe-v3-lts` (LTS version) to leverage the latest Zowe CLI and Zowe CLI plug-ins functionality.
 
-1. Update Zowe CLI. Open a command line window and issue the following command:
+### Updating from an npm online registry
+
+1. To update and install Zowe CLI core, open a command line window and issue the following command:
 
    ```
-   npm install -g @zowe/cli@zowe-v2-lts
+   npm install -g @zowe/cli@zowe-v3-lts
    ```
-2. Update Zowe plug-ins. Issue the following command to install all Zowe plug-ins:
+2. To update and install all Zowe plug-ins, open a command line window and issue the following command:
 
    ```
-   zowe plugins install @zowe/cics-for-zowe-cli@zowe-v2-lts @zowe/db2-for-zowe-cli@zowe-v2-lts @zowe/ims-for-zowe-cli@zowe-v2-lts @zowe/mq-for-zowe-cli@zowe-v2-lts @zowe/zos-ftp-for-zowe-cli@zowe-v2-lts
+   zowe plugins install @zowe/cics-for-zowe-cli@zowe-v3-lts @zowe/db2-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3lts @zowe/mq-for-zowe-cli@zowe-v3-lts 
    ```
-   **Note:** To install a subset of the plug-ins, remove the syntax for the plug-ins that you do not want to update. For example:
+   To install a subset of the plug-ins, remove the syntax for the plug-in(s) that you do not want to update. For example:
 
    ```
-   zowe plugins install @zowe/cics-for-zowe-cli@zowe-v2-lts @zowe/db2-for-zowe-cli@zowe-v2-lts
+   zowe plugins install @zowe/db2-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3-lts
    ```
 
-3. (Optional) Migrate your Zowe CLI profiles from your current installation to your V2 installation. Issue the following command:
+3. When updating from Zowe V1 to Zowe V3, migrate your Zowe CLI profiles from your current installation to your V3 installation:
 
    ```
    zowe config convert-profiles
    ```
-   Although you can run Zowe CLI V2 successfully using CLI V1 profiles, we strongly recommend using CLI V2 profiles.
-
-   **Note:** Profile data is backed up in case you want to revert the profiles to your previous Zowe CLI installation.
+   Profile data is backed up in case you want to revert the profiles to your previous Zowe CLI installation.
 
-4. (Optional) If you no longer require the profiles for your previous Zowe CLI installation, you can delete them. Issue the following command:
+4. When updating from Zowe V1 to Zowe V3, if you no longer require the profiles for your previous Zowe CLI installation, you can delete them:
 
    ```
    zowe config convert-profiles --delete
    ```
 
-   **Important:** We do not recommend deleting the profiles from your previous Zowe CLI installation until you have tested your V2 installation and are satisfied with its performance.
+   :::info Important
+   
+   We do not recommend deleting the profiles from your previous Zowe CLI installation until you have tested your V3 installation and are satisfied with its performance. **[is this still applicable for V3?]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
I guess I wouldn't say "satisfied with the performance, although performance-wise, V3 should be way better than V1.

Maybe just leave it as "...and are satisfied"